### PR TITLE
Fix objectstores3 testing

### PR DESCRIPTION
**Pull request by patrickjahns**
@mrow4a 
This should fix the test run for now - bucket is created beforehand

closes #1

### CODE REVIEW

**Comment by mrow4a** [comment]
@patrickjahns makes sense 👍 , please close the other PRs with this one. Thanks

File: .drone.yml [hunk 1]
```diff
@@ -22,22 +22,20 @@ pipeline:
...
     commands:
       - wait-for-it -t 300 scality:8000
 
+  create-bucket:
+    image: minio/mc
+    pull: true
+    environment:
+     - MC_HOSTS_scality=http://accessKey1:verySecretKey1@scality:8000
+    commands:
+     - mc mb scality/owncloud
```
Comment (DeepDiver1975) [inline .drone.yml]: Does this enable versioning?

Comment (mrow4a) [reply .drone.yml]: Good point, I also completely forgot about the fact that `occ s3` does create a bucket and enables versioning..

Comment (patrickjahns) [reply .drone.yml]: Hmm good catch - switched to aws-cli as poc here https://github.com/owncloud/smashbox-testing/pull/3

**Pull request merged**